src/utils/systemIntake.ts

```diff
@@ -39,3 +40,18 @@ export const translateRequestType = (requestType: RequestType) => {
       return '';
   }
 };
+
+/**
+ * Get the acronym representation of the component name
+ * @param componentToTranslate - component name string that needs to be converted to an acronym
+ */
+export const getAcronymForComponent = (componentToTranslate: string) => {
```

Comment:
I don't love this implementation, it feels like a very roundabout way of getting the acronym but I cant think of a better solution (besides adding the acronym to the intake itself)